### DISCUSSION TITLE
unable to retrieve the data from cassandra node though Stargate API

### DISCUSSION THREAD

**Discussion by Madhusudanarao123** [stargate/stargate#1595] [General] [answered]
Hi Team,

we are unable to retrieve the data from cassandra node though Stargate API

Thanks & Regards,
Madhu

**Comment by ivansenic** [comment]
@Madhusudanarao123 Can you give more context here, add exceptions or any other material that can help us in finding the issue. Thanks!

**Reply by Madhusudanarao123** [reply]
![image](https://user-images.githubusercontent.com/98576386/151740273-d7918417-989f-408a-bdc5-e77009910460.png)

using command
curl -s -L -v -X GET '{emp2}{users_keyspace}/{rkeyspace}/{rtable}?where=\{"firstname":\{"$in":\["sai","Madhu"\]\}\}' -H "X-Cassandra-Token: $AUTH_TOKEN" -H "Content-Type: application/json"

issue still not resolved, getting error. plz do need full

**Reply by ivansenic** [reply]
The error is `Could not resolve host`, you are not even reaching the Stargate, please have another look on the variables you are using to construct the URL.. Obviously there is a mistake there..

**Comment by Madhusudanarao123** [comment]
GCP level create 3 node cluster and separate vm install stargate, We are facing the issue like Stargate API server not able to communicating with Cassandra node for  Retrieve data.

PFB details..

Versions:
OS : Centos 7
Stargate Version : 1.0.41
Cassandra Version : 4.0.1

Using Command

curl -s -L -X GET '{emp2}{users_keyspace}/{rkeyspace}/{rtable}?where=\{"firstname":\{"$in":\["sai","Madhu"\]\}\}' -H "X-Cassandra-Token: $AUTH_TOKEN" -H "Content-Type: application/json"

![Stargate error](https://user-images.githubusercontent.com/98576386/151543216-0691994f-906b-4cda-bb35-d388477e1e82.png)

Command also excute but result not showing. But Other commands working fine (like create table/keyspace, alter , drop etc..)

**Reply by ivansenic** [reply]
Can you run with `curl` with the `-v` flag, so that we get a verbose output and see the resulting status code as well?

**Reply by Madhusudanarao123** [reply]
![Stargate error1](https://user-images.githubusercontent.com/98576386/151549637-e2d0ba40-e3fa-41b2-8fb1-a262991ccf73.png)
still getting same error, please help..

**Reply by Madhusudanarao123** [reply]
First command excute but output nil and as you said using -v getting error..

**Reply by Madhusudanarao123** [reply]
![image](https://user-images.githubusercontent.com/98576386/151551629-4b2b2cf3-7a5b-4b0f-a964-1f3e5e07c728.png)

Add -v also command excute and reslut nil

**Reply by ivansenic** [reply]
The `-X` need the parameter `GET`... so it should be `-v -X GET....`

**Comment by tatu-at-datastax** [comment, marked as answer]
I think this should have been an Issue, not a discussion; will try to convert.

**Comment by tatu-at-datastax** [comment]
Converted into #1598 -- please follow up there.
